CRM/Core/BAO/Mapping.php

```diff
@@ -115,7 +115,7 @@ public static function add($params) {
    *   Array of mapping names, keyed by id.
    */
   public static function getMappings($mappingType) {
-    $result = civicrm_api3('Mapping', 'get', array('mapping_type_id' => $mappingType, 'options' => array('limit' => 1, 'sort' => 'name')));
+    $result = civicrm_api3('Mapping', 'get', array('mapping_type_id' => $mappingType, 'options' => array('sort' => 'name')));
```

Comment:
This should have had the limit set to 0, not removed the limit. The mapping dropdown is now limited to the API default of 25. I'll open a new issue and PR.